src/fdk_rdf_parser/reference_data/extend_cpsvno_service.py

```diff
@@ -75,3 +77,21 @@ def extend_cv_output(
             )
             extended.append(output)
         return extended
+
+
+def extend_cv_evidence(
+    evidences: Optional[List[Evidence]], ref_data: PublicServiceReferenceData
+) -> Optional[List[Evidence]]:
+    if evidences is None:
+        return None
+    else:
+        extended: List[Evidence] = list()
+        for evidence in evidences:
+            evidence.dctType = extend_skos_code_list(
+                evidence.dctType, ref_data.evidence_type
+            )
+            evidence.language = extend_skos_code_list(
+                evidence.language, ref_data.linguisticsystem
+            )
+            extended.append(evidence)
+        return extended
```

Comment:
Dette er vel ekvivalent og mer forståelig?

```suggestion
        for evidence in evidences:
            evidence.dctType = extend_skos_code_list(
                evidence.dctType, ref_data.evidence_type
            )
            evidence.language = extend_skos_code_list(
                evidence.language, ref_data.linguisticsystem
            )
        return evidences
```

Reply:
Har opprettet issue for refaktorering